packages/dev/core/src/Materials/Background/backgroundMaterial.ts

```diff
@@ -137,6 +137,7 @@ class BackgroundMaterialDefines extends MaterialDefines implements IImageProcess
     public SKIPFINALCOLORCLAMP = false;
     public EXPOSURE = false;
     public MULTIVIEW = false;
+    public USEEXACTSRGBCONVERSIONS = false;

```

Comment:
To simplify things and not having to declare this variable in all materials (background, PBR, standard, etc), you should instead modify the `ThinEngine._getGlobalDefines` function.

It will simplify things a great deal because the define will be injected automatically for all shaders and you won't need to change any .ts files (except `thinEngine.ts`): only shader code has to be updated to take care of the define.

Reply:
Indeed, that is so much simpler! I had no idea about the global defines. Thank you for pointing that out.